### DISCUSSION TITLE
server: wordpress shortcode for embedding

### DISCUSSION THREAD

**Discussion by ku1ik** [asciinema/discussions#98] [Ideas]
https://twitter.com/byuhobbes/status/380426983772930049

**Comment by standage** [comment]
Just adding a comment to this ticket so I can see subsequent updates to it. Thanks!

**Comment by standage** [comment]
Regarding [your question](https://twitter.com/sickill/status/409727182647209984), take a look at the two links in my Twitter post above. WordPress.org refers to the blogging software that you can download and host yourself. The shortcode API is provided with WordPress.org, and new shortcodes would need to be developed on a locally hosted blog. Once you've developed and tested locally, you can submit a request to WordPress.com (their hosted counterpart) to get your shortcode approved in their hosted system.

I've never gone through this process, but I doubt it is very extensive. With the hard work of implementing the embedding already done, I assume the shortcode work will be much less in comparison.

**Comment by ku1ik** [comment]
@standage thx for taking your time and looking at this. I found some good resources describing the shortcode implementation process. Seems simple so I may give it a shot.

**Comment by ku1ik** [comment]
Just for reference, a Github gist shortcode implementation: http://wordpress.org/plugins/embed-github-gist/

**Comment by charlesjohnson** [comment]
+1 on this, I'd use the hell out of it.

**Comment by nogweii** [comment]
You might be able to avoid needing a separate wordpress plugin if oEmbed support is finished. (#125/#165)

**Comment by ku1ik** [comment]
@evaryont see this comment https://github.com/asciinema/asciinema.org/pull/165#discussion_r17213783

**Comment by ku1ik** [comment]
We have "image link" (PNG) embedding now, as well as support for oEmbed/Open Graph/Twitter Card: https://asciinema.org/docs/embedding
